alchemy/src/cloudflare/website.ts

```diff
@@ -81,7 +81,7 @@ export async function Website<B extends Bindings>(
           ? wrangler
           : (wrangler?.path ?? "wrangler.jsonc");
     const wranglerPath =
-      fileName && path.relative(cwd, path.join(cwd, fileName));
+      fileName && path.join(cwd, fileName);
```

Comment:
Not sure what this fixes? Relative is important so that state doesn't include absolute folders which are not portable between developers, for example.